hathor/conf/get_settings.py

```diff
@@ -15,10 +15,11 @@
 import importlib
 import os
 from types import ModuleType
+from typing import Optional
 
 from hathor.conf.settings import HathorSettings as Settings
 
-_config_file = None
+_config_file: Optional[str] = None
```

Comment:
I had to add this hint because for some reason `dmypy` was complaining about it, even if `mypy` was not.